opslevel/datasource_opslevel_domain_test.go

```diff
@@ -0,0 +1,29 @@
+package opslevel
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccDomainDataSource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Read testing
+			{
+				Config: testAccDomainDataSourceConfig,
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("data.opslevel_domain.test", "identifier", "my_domain"),
+				),
+			},
+		},
+	})
+}
+
+const testAccDomainDataSourceConfig = `
+data "opslevel_domain" "test" {
+  identifier = "my_domain"
```

Comment:
how does this work?  does this mean we need to maintain an opslevel account purely for testing the provider?

Reply:
yes